uw-research-computing/machine-learning-htc.md

```diff
@@ -105,4 +107,8 @@ B. Using Conda
 The Python package manager conda is a popular tool for installing and 
 managing machine learning tools.
 See [this guide](conda-installation) for information on how 
-to use conda to provide dependencies for CHTC jobs.
+to use conda to provide dependencies for CHTC jobs. 
+
+Note that when installing tensorflow using `conda`, it is important to install 
+not the generic `tensorflow` package, but `tensorflow-gpu`. This ensures that 
+the installation will include tensorflow's GPU capability. 
```

Comment:
```suggestion
Note that when installing TensorFlow using `conda`, it is important to install 
not the generic `tensorflow` package, but `tensorflow-gpu`. This ensures that 
the installation will include the `cudatoolkit` and `cudnn` dependencies
required for TensorFlow 's GPU capability. 
```

For our records (not this page), the [conda-forge channel](https://anaconda.org/conda-forge/tensorflow-gpu) and [default channel](https://anaconda.org/anaconda/tensorflow-gpu) both have `tensorflow-gpu`.  conda-forge will have newer versions available sooner.